.github/workflows/update-load-generator-image.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow uses GHA to build a Java project with Gradle 'load-generator', and build and push multi-arch docker image of this project to the Amazon ECR.
+
+name: Update Load-Generator Image
+
+# Should execute only one workflow run at a time
+concurrency:
+  group: build-and-push-load-generator-image
+
+on:
+  workflow_dispatch:
+  push:
+    branches: ['terraform']
+
+    # List of paths that should be considered when triggering the workflow
+    paths: ['load-generator/**']
+
+permissions:
+  id-token: write
+  contents: write
+
+jobs:
+  create-and-push-load-generator-image-on-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+          role-duration-seconds: 7200
+      - name: Login to Amazon ECR Public
+        id: login-ecr-public
+        uses: aws-actions/amazon-ecr-login@v1
+        with:
+          registry-type: public
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build with Gradle
+        run: ./gradlew :load-generator:build
+      - name: Build, tag, and push docker image to Amazon ECR Public
+        env:
+          REGISTRY: ${{ steps.login-ecr-public.outputs.registry }}
+          REGISTRY_ALIAS: aws-otel-test
+          REPOSITORY: aws-otel-load-generator
+          IMAGE_TAG: latest
```

Comment:
From what I noticed, All the current sample-apps are publishing [image with latest tag only](https://github.com/aws-observability/aws-otel-test-framework/blob/06496c5e679991d839b628afd737dcf28ce35214/terraform/imagebuild/main.tf#L61).
A git short sha can be added as a tag through the GHA workflow, but I couldn't find a solve on how to get that tag to become part of this [dockerfile path](https://github.com/aws-observability/aws-otel-test-framework/blob/06496c5e679991d839b628afd737dcf28ce35214/terraform/ec2_setup/variables.tf#L24) automatically. It will become a manual process to ensure that the latest/recent image is used. 
I couldn't find a existing setup in test-framework to reference this as well.

Reply:
Here is a similar example in [collector CI](https://github.com/aws-observability/aws-otel-collector/blob/daf6a4942f318cc31df927bf7e5cc92ba4cfe2dd/.github/workflows/CI.yml#L452). 

Is it possible to run `git rev-parse --short HEAD` in a subsequent job, store it as an output, and then reference that output in the list of tags?

Reply:
Hi. I have updated this. added multiple tags. In the follow up PR, I will pull this image using latest tag.